web-console/package.json

```diff
@@ -35,9 +35,11 @@
   },
   "dependencies": {
     "@blueprintjs/core": "^3.15.1",
+    "@types/d3": "^5.7.1",
```

Comment:
We've switched travis to using `npm ci` instead of `npm install` when building the web console package, so you'll need to either `npm install` and add the updated `package-lock.json` to git so that it can build successfully:

https://travis-ci.org/apache/incubator-druid/jobs/534049836#L1435